src/utils/constants.ts

```diff
@@ -5,7 +5,7 @@ dotenv.config()
 export const IVLE_KEY = process.env.REACT_APP_IVLE_KEY
 export const VERSION = process.env.REACT_APP_VERSION
 export const BACKEND_URL = process.env.REACT_APP_BACKEND_URL
-export const USE_BACKEND = process.env.REACT_APP_USE_BACKEND
```

Comment:
Please change this back to the original value, you can set this variable using the `.env` file. If you don't have one, you can make a copy of it from `.env.example`

Reply:
Oh man. Yup am aware of `.env`. 

I changed it to try something and accidentally pushed it in (sry)

I changed it back